Antidote/CallsManager.m

```diff
@@ -209,6 +228,7 @@ - (void)activeCallSpeakerButtonPressed:(ActiveCallViewController *)controller
             controller.speakerSelected = YES;
         }
         else if (error.code == OCTToxAVErrorControlInvaldTransition) {
+            DDLogWarn(@"%@, sendCallControl error:%@", self, error.localizedFailureReason);
```

Comment:
If code isn't equal to InvalidTransition there will be no log. It would be better to put this log after `else if` block.